public/data/PriyanshuIndra.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Priyanshu Indra",
+    "bio": "Student",
+    "avatar": "https://avatars.githubusercontent.com/u/74089982?v=4",
+    "links": [
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/PriyanshuIndra4",
+            "icon": "twitter"
+        }
+    ]
+}
```

Comment:
```suggestion
}

```

Reply:
I don't understand by revert !! do you mean deleting brackets ??